suzieq/poller/worker/nodes/node.py

```diff
@@ -1568,6 +1568,15 @@ def _extract_nos_version(self, data: str) -> None:
 class LinuxNode(CumulusNode):
     '''Linux server node'''
 
+    async def _rest_connect(self):
```

Comment:
Since its inheriting from CumulusNode which already rejected REST, do we need this too? Or is this just additional protection?

Reply:
In theory it shouldn't be needed but If we don't override the methods also in the LinuxNode pylint complains, I believe because since the methods in Cumulus raise a NotImplementedError pylint still considers the methods as abstract

Reply:
Thanks